setup.py

```diff
@@ -47,6 +47,7 @@ def read(fname):
         'boto >= 2.32',
         'httpretty==0.8.6',
         'bz2file',
+        'requests',
```

Comment:
Oh I see, it's one of the recent additions by @ziky90 , I didn't have that pulled locally yet. Cheers!